### ISSUE TITLE
Generate image of specific height-Width

### ISSUE THREAD

**Issue by dibs439** [mcastorina/graph-cli#29]
Hi,

I wanted to control the size of the output bar diagram. Now, when I use the param --figsize=634x117 the command line params returns a Out of memory error. However, when I do not add this in my command the out file generate and with a greater resolution of 1600x1000. Can you please help.

Thanks

**Comment by mcastorina** [comment]
Hmm.. that's really strange. I did a quick experiment to confirm:

```
> graph samples/cosine.csv --figsize=160x100 -o test2.png
> exiftool test2.png
ExifTool Version Number         : 11.50
File Name                       : test2.png
Directory                       : .
File Size                       : 1087 kB
File Modification Date/Time     : 2020:06:14 19:02:04-05:00
File Access Date/Time           : 2020:06:14 19:02:02-05:00
File Inode Change Date/Time     : 2020:06:14 19:02:04-05:00
File Permissions                : rw-r--r--
File Type                       : PNG
File Type Extension             : png
MIME Type                       : image/png
Image Width                     : 16000
Image Height                    : 10000
Bit Depth                       : 8
Color Type                      : RGB with Alpha
Compression                     : Deflate/Inflate
Filter                          : Adaptive
Interlace                       : Noninterlaced
Significant Bits                : 8 8 8 8
Pixels Per Unit X               : 3937
Pixels Per Unit Y               : 3937
Pixel Units                     : meters
Software                        : matplotlib version3.1.1, http://matplotlib.org/
Image Size                      : 16000x10000
Megapixels                      : 160.0
```

**Comment by mcastorina** [comment]
Now that I think about it though, `figsize` is input directly to matplotlib and the parameter is not in pixel units. I see how that can be confusing for a user.

On my system, "pixels per unit X" and "pixels per unit Y" seem consistently 3937, so if you're looking for pixel dimensions `634x117` I would try `0.16x0.03`

**Comment by mcastorina** [comment]
Actually, `--figsize` doesn't take float:

```
Traceback (most recent call last):
  File "/usr/local/bin/graph", line 8, in <module>
    sys.exit(main.main())
  File "/usr/local/lib/python3.7/site-packages/graph_cli/main.py", line 8, in main
    args = options.parse_args()
  File "/usr/local/lib/python3.7/site-packages/graph_cli/options.py", line 375, in parse_args
    return validate_args(parser.parse_args())
  File "/usr/local/lib/python3.7/site-packages/graph_cli/options.py", line 60, in validate_args
    fill_global_args(args, df)
  File "/usr/local/lib/python3.7/site-packages/graph_cli/options.py", line 146, in fill_global_args
    args.figsize = (tuple(map(int, args.figsize.split('x'))), True)
ValueError: invalid literal for int() with base 10: '0.16'
```

I'll mark this as a bug with the fix being convert pixels to whatever unit matplotlib uses.

**Comment by mcastorina** [comment]
Fixed in 13a3ccb966fdd83f99740d4f2a9458d33012f0c3 and uploaded version 0.1.5 which you can install from pip.
